install_config/aggregate_logging.adoc

```diff
@@ -1345,6 +1338,75 @@ from that date. If you want to be exact with Curator, it is best to use days
 (for example, `delete: days: 30`).
 ====
 
+[[aggregate-logging-using-curator-actions-file]]
+==== Using the Curator Actions File
+
+Setting the {product-title} custom configuration file format ensures internal
+indices are not mistakenly deleted.
+
+To use the *actions file*, add an exclude rule to your configuration to retain
+these indices. You must manually add all of the required patterns.
+
+----
+config.yaml: |
```

Comment:
@josefkarasek 
the action file should be put under actions.yaml, not config.yaml